connectors/sources/github.py

```diff
@@ -1397,6 +1643,46 @@ def is_previous_repo(self, repo_name):
         self.prev_repos.append(repo_name)
         return False
 
+    def _decorate_with_access_control(self, document, access_control):
+        if self._dls_enabled():
+            document[ACCESS_CONTROL] = list(
+                set(document.get(ACCESS_CONTROL, []) + access_control)
+            )
+        return document
+
+    async def _fetch_access_control(self, repo_name):
+        owner, repo = self.github_client.get_repo_details(repo_name)
+        collaborator_variables = {
+            "orgName": owner,
+            "repoName": repo,
+            "cursor": None,
+        }
+        access_control = []
+        if len(self.members) <= 0:
+            async for user in self.github_client._fetch_all_members():
+                self.members.add(user.get("id"))
```

Comment:
What happens if `id` is `None`? Should we skip it then?

Reply:
As mentioned above I don't think we can get empty ID. In our testing we did not faced this scenario.